src/js/stores/search.js

```diff
@@ -231,18 +239,26 @@ define(function (require, exports, module) {
                         pathUtil.getShortestUniquePaths(ancestors).toJS() : ancestors.toJS();
 
                 var itemMap = items.map(function (item, index) {
-                    var newPathInfo = shortenedPaths[index] || "";
+                    var newPathInfo = shortenedPaths[index] || "",
+                        name = item.name,
+                        style;
                     // Don't show the path info if it is just the same as the item name 
-                    if (item.name === newPathInfo) {
+                    if (name === newPathInfo) {
                         newPathInfo = "";
                     }
 
+                    if (name === "") {
+                        name = "Untitled";
+                        style = { "font-style": "italic" };
```

Comment:
This should be `fontStyle`: https://facebook.github.io/react/tips/inline-styles.html